### PR TITLE
add genznodes service

### PR DESCRIPTION
**Pull request by kii-awesome**

### CODE REVIEW

File: peers_seeds_and_services.md [hunk 1]
```diff
@@ -291,7 +309,11 @@ f0e6c86d769bf5c52f78e01864091690e731643f@bitcanna-seed.panthea.eu:37656
 			{
 				"address": "https://bitcanna-grpc.lavenderfive.com:443/",
 				"provider": "Lavender.Five Nodes 🐝"
-			}
+			},
+			{
+				"address": "bitcanna-grpc.genznodes.dev:32090",
+				"provider": "genznodes"
+			},
```
Comment (RaulBernal) [inline peers_seeds_and_services.md]: I think it should ends it `}` 
Can you take a look @Genz22 ? Thanks

**Review by RaulBernal — CHANGES_REQUESTED** [review]
Please check line 316.

File: peers_seeds_and_services.md (hunk 1 above)
Comment (kii-awesome) [reply peers_seeds_and_services.md]: fixed

**Review by RaulBernal — APPROVED** [review]
Thanks a lot :)

**Pull request merged**